src/App.js

```diff
@@ -1,25 +1,65 @@
 import React from 'react';
-
+import { Clock } from './components/Clock';
 import './App.scss';
 
-const App = () => {
-  setInterval(() => {
-    const date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
-};
-
-export default App;
+export class App extends React.Component {
+  state = {
+    isClockVisible: true,
+    clockName: Math.random(),
+    clockNewName: Math.random(),
+  }
+
+  genereteRandomName = () => {
+    this.setState({ clockNewName: Math.random() });
+    // eslint-disable-next-line react/no-access-state-in-setstate
+    this.setState({ clockName: this.state.clockNewName });
```

Comment:
```suggestion
    // eslint-disable-next-line react/no-access-state-in-setstate
    this.setState({ clockName: Math.random() });
```
To get access to the previous state you should use functional setState. That's why you got linter error. don't disable es-lint